x/evm/types/command.go

```diff
@@ -62,7 +70,7 @@ func NewBurnTokenCommand(chainID sdk.Int, keyID multisig.KeyID, height int64, bu
 
 	return Command{
 		ID:         NewCommandID(append(burnerInfo.Salt.Bytes(), heightBytes...), chainID),
-		Command:    axelarGatewayCommandBurnToken,
+		Command:    COMMAND_TYPE_BURN_TOKEN,
```

Comment:
Does this change not require migration though?

Reply:
oh damn you are right! I did it differently before, but now it's a different data type